src/Core/Deprecated/Undeprecator.js

```diff
@@ -29,27 +29,48 @@ export const deprecatedParsingOptionsToNewOne = (options) => {
         newOptions.out.mergeFeatures = options.mergeFeatures;
         newOptions.out.withNormal = options.withNormal;
         newOptions.out.withAltitude = options.withAltitude;
```

Comment:
Is there a point in copying `options.withNormal` and `options.withAltitude` parameters in `newOptions` since both these parameters are deprecated ?
Maybe you could instead remove lines 30 and 31, and replace line 32 by : 
```js
if (options.withAltitude && options.withNormal)
```